release/mk-release-series.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+
+# Location of the repository checkouts
+vcsdir=`cat ~/.code-vcs-path`
```

Comment:
A common idiom for these things is to source a single file setting some variables.
Among the benefits, it's a single location, and it allows full shell syntax, variable expansion, and comments

The calling pattern is usually:
```
test -e ~/.release-settings && . ~/.release-settings
```
can we adopt that style instead?

Reply:
+1. And if we are going to pollute home directory with these things, let's use Squid-specific names like squid-release-settings or squid-release-configuration (with a leading dot if really necessary).

Said that, if my [assumption](https://github.com/squid-cache/ci/pull/8#pullrequestreview-1276438587) that this PR just imports what we already have is correct, then these script improvements do not belong to _this_ PR: We should not mix import of an old script with new improvements of that old script. If my assumption is wrong, then it would be best to import the old version in this PR and then created PR(s) dedicated to ongoing improvements.

Reply:
Well yes. We could have a better way to pull in the environment layout details to all these release scripts. That may be better as a followup to switch them all at once (and document how it works).

Reply:
Doing this in PR #7. Followups there please.